examples/package.json

```diff
@@ -0,0 +1,5 @@
+{
+    "dependencies": {
+        "revai-node-sdk": "^2.0.1"
```

Comment:
We should make this `"revai-node-sdk": "*"`. This way they will always get the latest version of the sdk